Mathlib/Algebra/Periodic.lean

```diff
@@ -3,14 +3,11 @@ Copyright (c) 2021 Benjamin Davidson. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Benjamin Davidson
 -/
-import Mathlib.Algebra.BigOperators.Basic
 import Mathlib.Algebra.Field.Opposite
-import Mathlib.Algebra.Module.Basic
 import Mathlib.Algebra.Order.Archimedean
-import Mathlib.Data.Int.Parity
 import Mathlib.GroupTheory.Coset
-import Mathlib.GroupTheory.Subgroup.ZPowers
 import Mathlib.GroupTheory.Submonoid.Membership
+import Mathlib.GroupTheory.Subgroup.ZPowers
```

Comment:
Why did you move this down a line? I think the list used to be alphabetically ordered... for whatever that's worth.

Reply:
Because it was late and I thought it'd be ordered this way? I don't remember; fixed.